src/Instrumentation/Laravel/src/LaravelInstrumentation.php

```diff
@@ -4,8 +4,8 @@
 
 namespace OpenTelemetry\Contrib\Instrumentation\Laravel;
 
-use Illuminate\Foundation\Application;
-use Illuminate\Foundation\Http\Kernel;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
Each Watcher is already typed against `\Illuminate\Contracts\Foundation\Application`, so this maintains that type for consistency.